/**:
  ros__parameters:
    server: ntp.ubuntu.com
    offset_warn: 0.1
    offset_error: 5.0
